csharp/AppEncryption/AppEncryption.Tests/AppEncryption/SessionFactoryTest.cs

```diff
@@ -454,7 +454,7 @@ private void TestSessionCacheGetSessionWithMaxSessionReachedButStillUsedShouldNo
             }
         }
 
-        [Fact]
+        [Fact(Skip = "Fails randomly during CI")]
         private void TestSessionCacheMultiThreadedSameSessionNoEviction()
```

Comment:
I think we should log a separate issue for these tests and fix them there. IIRC at least 1 of these tests fails/failed intermittently on CircleCI as well. That makes me think that the issue lies in the test not CircleCI or GH Actions?

Reply:
I just wanted to ensure these didn't fall by the wayside, so that works for me. Have these issues already been logged? If so, please drop in a ref to this PR.